l10n.py

```diff
@@ -48,235 +63,325 @@
     GetNumberFormatEx.restype = ctypes.c_int
 
 
-class Translations(object):
+class _Translations:
 
-    FALLBACK = 'en'	# strings in this code are in English
+    FALLBACK = 'en'  # strings in this code are in English
     FALLBACK_NAME = 'English'
 
-    TRANS_RE   = re.compile(r'\s*"((?:[^"]|(?:\"))+)"\s*=\s*"((?:[^"]|(?:\"))+)"\s*;\s*$')
+    TRANS_RE = re.compile(r'\s*"((?:[^"]|(?:\"))+)"\s*=\s*"((?:[^"]|(?:\"))+)"\s*;\s*$')
     COMMENT_RE = re.compile(r'\s*/\*.*\*/\s*$')
 
+    def __init__(self) -> None:
+        self.translations: Dict[Optional[str], Dict[str, str]] = {None: {}}
 
-    def __init__(self):
-        self.translations = { None: {} }
+    def install_dummy(self) -> None:
+        """
+        Install a dummy translation function.
 
-    def install_dummy(self):
-        # For when translation is not desired or not available
-        self.translations = { None: {} }
-        builtins.__dict__['_'] = lambda x: str(x).replace(r'\"', u'"').replace(u'{CR}', u'\n')	# Promote strings to Unicode for consistency
+        Use when translation is not desired or not available
+        """
+        self.translations = {None: {}}
+        # Promote strings to Unicode for consistency
```

Comment:
Misleading comment, as we're no longer doing that.  What the next line does is some replacements on quotes and the like.

Reply:
Removed the comment, what the below does is reasonably self explanitory